static/app/components/interactionStateLayer.tsx

```diff
@@ -79,4 +85,8 @@ const InteractionStateLayer = styled(
   }
 `;
 
+InteractionStateLayer.defaultProps = {
+  hasSelectedBackground: true,
+};
```

Comment:
This is how default props have to be defined apparently 🫠

Reply:
Oh I see, because it's a styled component I guess